docs/streams/developer-guide/config-streams.html

```diff
@@ -80,6 +80,7 @@
               <li><a class="reference internal" href="#deserialization-exception-handler" id="id7">deserialization.exception.handler</a></li>
               <li><a class="reference internal" href="#enable-metrics-push" id="id43">enable.metrics.push</a></li>
               <li><a class="reference internal" href="#ensure-explicit-internal-resource-naming" id="id46">ensure.explicit.internal.resource.naming</a></li>
+              <li><a class="reference internal" href="#group-protocol" id="id47">group.protocol</a></li>
```

Comment:
Random thought: I just checked and priority is set to MEDIUM in `StreamsConfig`. Should we reduce it to `LOW` for EA?

Reply:
We could do it in a separate PR if we have time. I don't want to touch the code in this pr.